katz/src/main/kotlin/katz/typeclasses/Functor.kt

```diff
@@ -17,5 +17,11 @@
 package katz
 
 interface Functor<F> {
+
     fun <A, B> map(fa: HK<F, A>, f: (A) -> B): HK<F, B>
+
+    fun <A, B> lift(f: (A) -> B): (HK<F, A>) -> HK<F, B> =
```

Comment:
In this case `lift` is something that instances may override. Same with `map` and other similar combinators. In the case of `or` in `FunctionK` is different because that is a utility function frequently used to build interpreters